pkg/controller/controller.go

```diff
@@ -455,7 +464,15 @@ func (ctrl *resizeController) markPVCResizeInProgress(pvc *v1.PersistentVolumeCl
 	newPVC := pvc.DeepCopy()
 	newPVC.Status.Conditions = util.MergeResizeConditionsOfPVC(newPVC.Status.Conditions,
 		[]v1.PersistentVolumeClaimCondition{progressCondition})
-	return util.PatchPVCStatus(pvc, newPVC, ctrl.kubeClient)
+	updatedPVC, err := util.PatchPVCStatus(pvc, newPVC, ctrl.kubeClient)
+	if err != nil {
+		return nil, err
+	}
+	err = ctrl.claims.Update(updatedPVC)
+	if err != nil {
+		return nil, fmt.Errorf("error updating PVC %s in local cache: %v", util.PVCKey(newPVC), err)
```

Comment:
Do we want to fail the entire operation if the local cache fails to update?

Reply:
In the worst case when updating local cache failed and the informer cache did not update itself (from PVC events) while operation was retried - the test will flake.

So, it should be fine I think.